2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -839,7 +839,7 @@
  {
   "input": "By the way, if you want to stay up to date with new videos and also any other projects that I put out there like the Summer of Math Exposition, there is a mailing list.",
   "model": "nmt",
-  "translatedText": "Por cierto, si desea mantenerse actualizado con nuevos videos y también con cualquier otro proyecto que publique, como la Exposición de Verano de Matemáticas, hay una lista de correo.",
+  "translatedText": "Por cierto, si quieres mantenerte actualizado con nuevos vídeos y también con cualquier otro proyecto que publique, como la Exposición de Matemáticas de Verano, hay una lista de correo.",
```

This is a more general comment, but I am also curious about the usage of "we" throughtough the translation (while in English they used "you"), but then in sentences like this you switch back to "you". You could very well say:

```suggestion
  "translatedText": "Por cierto, si quieren mantenerse al tanto de nuevos vídeos y también de cualquier otro proyecto que publiquemos, como la Exposición de Matemáticas de Verano, tenemos una lista de correos.",
```

I would also change "hay" to "tenemos" (or "tengo"), and use "(mantenerse) al tanto de" instead of "(mantenerse) actualizado con". If you want to keep the "you" conjugations:

```suggestion
  "translatedText": "Por cierto, si quieres mantenerte al tanto de nuevos vídeos y también de cualquier otro proyecto que publique, como la Exposición de Matemáticas de Verano, tenemos una lista de correos.",
```